x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/args.test.ts

```diff
@@ -51,7 +51,7 @@ describe('headless webgl arm mac workaround', () => {
     // please double-check that the --use-angle flag is still needed for arm macs
     // instead of --use-angle you may need --enable-gpu
     expect(getChromiumPackage().binaryChecksum).toBe(
-      'e5d4ccdbf3b66532c7c0973be2d5bbd3189079646ced55fe4db61d8e7e7bfc7e'
```

Comment:
This test is here for us to check whether the `--enable-gpu` flag will work on Mac ARM, rather than using `--use-angle`. I just gave it a check on my Mac and it seems to work! 

If it works for you, let's get rid of this test -- it adds to the burden of backporting these updates to 7.17.

Since this PR is urgent, we can take this as a comment to consider for a followup.

Reply:
Hey @tsullivan thanks for the callout here, I tested on my machine `--enable-gpu` now works in place of passing `--use-angle`. The resolution is also verifiable [here](https://issues.chromium.org/issues/40256775#comment7).